Source/SideMenuController.swift

```diff
@@ -132,7 +132,7 @@ public extension SideMenuController {
     // MARK: Public
     
     open weak var delegate: SideMenuControllerDelegate?
-    open static var preferences: Preferences = Preferences()
+    public static var preferences: Preferences = Preferences()
```

Comment:
No more `open` in Swift 5?

Reply:
Apparently, I think it has to do with declaring it static. So `open` isn't gone but it crashes with static I think.